iib/workers/tasks/git_utils.py

```diff
@@ -310,6 +310,7 @@ def configure_git_user(
     :param str user_name: User name for local Git repo.
     :param str email_address: Email address for local Git repo.
     """
+    log.info("Configuring git user.name and user.email for local Git repo %s.", local_repo_path)
     run_cmd(
         ["git", "-C", local_repo_path, "config", "--local", "user.name", str(user_name)],
```

Comment:
Why we dont use `configure_git_user` here?

Reply:
@MichalZelenak Instead of configuring the git user here, we decided to define them during deployment as GIT_* env vars.